src/components/range/range.ts

```diff
@@ -360,6 +365,9 @@ export class Range extends Ion implements AfterViewInit, ControlValueAccessor, O
       return false;
     }
 
+    // trigger ionFocus event
+    this.ionFocus.emit(this);
```

Comment:
let's move this at the bottom of the function just to make sure we don't break anything!